docs/development/virtual_environments.rst

```diff
@@ -160,7 +166,71 @@ virtualized, rather than running on hardware. You will need to
 ``ANSIBLE_ARGS="--skip-tags validate"`` to skip the tasks that prevent the prod
 playbook from running with Vagrant-specific info.
 
-To create only the prod servers, run:
+You can provision production VMs from an Admin Workstation (most realistic),
+or from your host. Instructions for both follow.
+
+.. _prod_install_from_tails:
+
+Install from an Admin Workstation VM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In SecureDrop, admin tasks are performed from a Tails *Admin Workstation*.
+You should configure a Tails VM in order to install the SecureDrop production VMs
+by following the instructions in the :ref:`Virtualizing Tails <virtualizing_tails>`
+guide.
+
+Once you're prepared the *Admin Workstation*, you can start each VM:
+
+.. code:: sh
+
+  vagrant up --no-provision /prod/
+
+At this point you should be able to SSH into both ``app-prod`` and ``mon-prod``.
+From here you can follow the :ref:`server configuration instructions
+<test_connectivity>` to test connectivity and prepare the servers.
+
+.. note:: If you have trouble SSHing to the servers from Ansible, remember
+          to remove any old ATHS files in ``install_files/ansible-base``.
+
+Now from your Admin workstation:
+
+.. code:: sh
+
+  cd ~/Persistent/securedrop
+  ./securedrop-admin setup
+  ./securedrop-admin sdconfig
+  ./securedrop-admin install
+
+.. note:: The sudo password for the ``app-prod`` and ``mon-prod`` servers is by
+          default ``vagrant``.
+
+After install you can configure your Admin Workstation to SSH into each VM via:
+
+.. code:: sh
+
+  ./securedrop-admin tailsconfig
+
+Install from Host
+~~~~~~~~~~~~~~~~~
+
+If you are not virtualizing Tails, you can manually modify ``site-specific``,
+and then provision the machines. You should set the following options in
+``site-specific``:
+
+.. code:: sh
+
+  ssh_users: "vagrant"
+  monitor_ip: "10.0.1.5"
+  monitor_hostname: "mon-prod"
+  app_hostname: "app-prod"
+  app_ip: "10.0.1.4"
+
+Note that you will also need to generate Submission and OSSEC PGP public keys,
+and provide email credentials to send emails to. Refer to
+:ref:`this document on configuring prod-like secrets<configure_securedrop>`
+for more details on those steps.
```

Comment:
I'm on the fence whether we should support non-Tails-administered prod VMs. Seems like lack of Tails should require testing in staging. Thoughts?

Reply:
Best discussed over in #2203.